test/base.test.js

```diff
@@ -99,7 +101,6 @@ test('should set session cookie using the default cookie name', async (t) => {
     fastify.addHook('onRequest', (request, reply, done) => {
       request.sessionStore.set('Qk_XT2K7-clT-x1tVvoY6tIQ83iP72KN', {
         expires: Date.now() + 1000,
-        sessionId: 'Qk_XT2K7-clT-x1tVvoY6tIQ83iP72KN',
```

Comment:
there's only a getter, no setter. Probably makes this a breaking change even though _changing_ the session id has never been supported I think?

It's possibly breaking regardless as somebody have a custom store that relies on this data being present? not sure

Reply:
I could revert this after updating the allowlist in `this[addDataToSession]`. It's now ignored instead